test/integration/local/services/message-receiver.test.js

```diff
@@ -3,7 +3,7 @@ const MessageSender = require('../../../../server/services/messaging/message-sen
 const config = require('../../../../server/config')
 
 describe('message receiver', () => {
-  const message = { content: 'hello', claimId: 'claimId' }
+  const message = { content: 'hello', claimId: 'testClaimId' }
```

Comment:
Strictly speaking there is no need for `claimId` as the receiver uses the action supplied by the test rather than the one in the message service that would be expecting the property. It should be ok to remove the property from the message.

Reply:
On further inspection, the test itself isn't correct, as it asserting on a non existent property, so it will always be undefined.
I removed the extra param and asserted on the `content` property.